command.ts

```diff
@@ -933,6 +937,8 @@ XRANGE somestream - +
     increment: number,
     member: RedisValue,
   ): Promise<BulkString>;
+  zinter(keys: string[], opts?: ZInterOpts): Promise<Raw[]>;
+  zinter(key_weights: [string, number][], opts?: ZInterOpts): Promise<Raw[]>;
```

Comment:
Could you add the following overload signature to support `Record<string, number>`?

```suggestion
  zinter(key_weights: [string, number][], opts?: ZInterOpts): Promise<Raw[]>;
  zinter(key_weights: Record<string, number>, opts?: ZInterOpts): Promise<Raw[]>;
```